packages/vuetify/src/labs/VCalendar/VCalendarIntervalEvent.tsx

```diff
@@ -0,0 +1,72 @@
+// Components
+import { VSheet } from '@/components/VSheet'
+
+// Composables
+import { useDate } from '@/composables/date'
+
+// Utilities
+import { genericComponent, useRender } from '@/util'
+
+export const VCalendarIntervalEvent = genericComponent()({
+  name: 'VCalendarIntervalEvent',
+
+  props: {
+    allDay: Boolean,
+    interval: Object,
+    intervalDivisions: {
+      type: Number,
+      required: true,
+    },
+    intervalDuration: {
+      type: Number,
+      required: true,
+    },
+    intervalHeight: {
+      type: Number,
+      required: true,
+    },
+    event: Object,
+  },
+
+  setup (props, { emit, slots }) {
+    const adapter = useDate()
+    const calcHeight = () => {
```

Comment:
This determines the height of the event div within the interval. If the event start time occurs within the interval, but not equal with the start, it will return the adjusted height and margin necessary to better represent the event start time.